### ISSUE TITLE
Investigate diskmanager error

### ISSUE THREAD

**Issue by aebling** [JNPRAutomate/space-packer#5]
When creating the second disk there are configuration errors however the vmdk gets created. Would be nice if we can clear these errors however the flags look valid.

**Comment by routelastresort** [comment]
The missing config file is normal / misleading.  One concern is how the 100GB disk gets mounted by Space - I haven't looked into that too much.

**Comment by routelastresort** [comment]
`VixDiskLib: Invalid configuration file parameter. Failed to read configuration file.` isn't going away w/o modifying configuration files outside of the space-packer directory - unless you find a better answer.  The command-line flags don't support anything to fix it - maybe it takes a configuration from the cwd?

**Comment by aebling** [comment]
This may need to be modified in that in the current release of 14.1 the recommended method is to create multiple disk resources and then expand the partitions in space one at a time via the cli/menu. Partitioning as follows:
40gb for /var
25gb for /var/log
15gb for /tmp
20gb for /

This is done via post install in the space cli menu to expand partitions. If we follow this method the current provisioning will not work.

**Comment by aebling** [comment]
Sorry didn't mean to close the issue. It is still open.

**Comment by routelastresort** [comment]
Hmmmmm - so each one is a separate disk, e.g.:
- 40gb for /var = disk2
- 25gb for /var/log = disk3
- 15gb for /tmp = disk4
- 20gb for / = disk5

This would be trivial to support, if that's what the need is.  Have you had a chance to briefly check out the other 14.1 changes, and see if I missed anything big?  I'm thinking the IP addressing will be the interesting part.

**Comment by aebling** [comment]
Correct. In the vm build I have I have a total of 5 disks. It explicitly stated in the deployment guide that we could not create 100gb and carve it up.
